operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -1563,6 +1586,34 @@ func (rc *ReconciliationContext) findStartedNotReadyNodes() (bool, error) {
 	return false, nil
 }
 
+func (rc *ReconciliationContext) copyPodCredentials(pod *corev1.Pod, jksBlob []byte) error {
```

Comment:
This function was brought in from a merge of master into my branch.  Was this a failed merge conflict resolution?  Should I delete this function? @johntrimble ?

Reply:
I don't know what that is. It looks like something from one of Eldon's PRs.

Reply:
I will remove it.